styles/styles.css

```diff
@@ -0,0 +1,218 @@
+body {
+  font-family: sans-serif;
+}
+
+header {
+  background: url("../images/header-image.jpg") no-repeat top;
+  color: #81B741;
+  font-size: 1.25rem;
+  height: 20rem;
+  padding-top: 3rem;
+  box-sizing: border-box;
+}
+
+header img.logo {
+  width: 100%;
+}
+
+header .cart, .logo-container {
+  display: inline-block;
+
+}
+
+header .logo, .motto {
+  padding-left: 2rem;
+}
+
+@media (min-width: 840px) {
+  header .logo, .motto {
+    padding-left: 10rem;
+  }
+}
+
+.cart {
+  float: right;
+  padding: 1.875rem;
+}
+
+.cart img {
+  height: 1.25rem;
+}
+
+.cart span {
+  color: white;
+  font-size: .875rem;
+}
+
+.cart button {
+  background-color: #81B741;
+  color: #FFF;
+  display: block;
+}
+
+h1 {
+  font-size: 1.5625rem;
+}
+
+nav.global {
+  background-color: #E8E8E8;
+  color: #323232;
+  padding: 30px;
+  font-weight: bold;
+  display: flex;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+}
+
+nav a {
+  color: #323232;
+  text-decoration: none;
+  padding: 0 .9375rem;
```

Comment:
Minor nitpick, but it's nice to give values that are less than 0 a leading 0. in your case you code could look like

```
nav a {
  padding: 0 0.9375rem;
|
```

Especially with the other 0 it just looks a little clunky.

Reply:
Good point. I will update that! Thank you!